icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -1174,10 +1173,24 @@ class U_I18N_API Usage : public UMemory {
     // Allow skeleton generation code to access private members.
     friend class impl::GeneratorHelpers;
 
-    // Allow MacroProps/MicroProps to initialize empty instances.
-    friend struct impl::MacroProps;
-
 #endif // U_HIDE_INTERNAL_API
+
+  private:
+    /** @internal */
+    UErrorCode fError;
```

Comment:
Please keep fields with fields, functions with functions, and friends with friends.  Also, remove the second redundant `private:`. If `#endif // U_HIDE_INTERNAL_API` is causing problems, just move it up above the first `private:`.

Reply:
Done. Not squashed yet.